inst/tutorials/r_and_rstudio_ws/r_and_rstudio_ws.Rmd

```diff
@@ -300,19 +304,20 @@ As you can see, when you multiply two vectors with the same length, R multiplies
 ### Data frames
 Whereas a vector is a one-dimensional structure, *data frames* are two-dimensional. You can think of a data frame as a collection of vectors. Data frames are one of R's most essential data structures. They organise data into a tabular format with rows and columns. 
 
-If you have some data from an experiment or study, you will typically save it as a table in a format such as `.csv`, `.txt`, or `.xlsx`. When you load those files into R, you will create a data frame object. At the end of this tutorial (section *Using R in your own computer*) you will learn how to load a data file stored in your computer into R.
+If you have some data from an experiment or study, you will typically save it as a table in a format such as `.csv`, `.txt`, or `.xlsx`. When you load those files into R, you will create a data frame object. At the end of this tutorial (section *Using R in your own computer*) you will learn how to load a data file stored in your computer into R. Instead of loading data from your computer, you can also use an online data set.
+
+``` {r child = system.file("resources/markdown", "data_description.Rmd", package = "educer")}
+```
 
-Instead of loading data from your computer, you can also use an online data set: we will use [this one](https://raw.githubusercontent.com/EDUCE-UBC/workshops_access/master/intro_R_2hr/data.csv), which is a `.csv` file hosted on our [GitHub](https://github.com/EDUCE-UBC/workshops_access). Click on the link and examine the format of the data. Notice that the first row consists of column names, that each column corresponds to a data variable, and that each row corresponds to one observation.
+The [data file](https://raw.githubusercontent.com/EDUCE-UBC/workshops_access/master/intro_R_2hr/data.csv) is a `.csv` file hosted on our [GitHub](https://github.com/EDUCE-UBC/workshops_access). Click on the link and examine the format of the data. Notice that the first row consists of column names, that each column corresponds to a data variable, and that each row corresponds to one observation.
```

Comment:
Nice job. I'm learning Tensorflow in Python right now and something that's been very handy is examining your data structures often. I'm sure the students will appreciate this.